quic/s2n-quic-core/src/connection/id.rs

```diff
@@ -276,6 +276,26 @@ impl TryFrom<LocalId> for InitialId {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Classification {
+    /// The connection ID was chosen by the client
```

Comment:
```suggestion
    /// The connection ID was chosen by the peer client
```

Reply:
Not sure what you mean by this suggestion... I wanted to keep this enum generic enough that it could be used by either endpoint. So it might not always be a peer. But it will always be client-chosen.

Reply:
Ah, I was thinking it was only used for classifying a connection ID seen on an incoming datagram

Reply:
It currently is limited to that. But I'd like it to be flexible for future uses, too.